tests/perforce_utils.py

```diff
@@ -194,6 +194,21 @@ def perforce_workspace(request, perforce_connection, tmpdir):
         change["Description"] = "Test submit"
         p4.run_submit(change)
 
+        permissions = p4.fetch_protect()
+        permissions['Protections'] = [
+            'write user * * //...',
```

Comment:
Are the first three items default?
If yes, please explicitly mark them in comment.